_posts/2020-04-07-trying-out-julia.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: Trying out Julia programming language
+author: hnybom
+excerpt: >
+  Julia is a programming language with it's roots in the mathematics. The language itself is increasingly gaining momentum and that combined with the boom were seeing in data science demand makes it a very interesting subject. This post is about my journey starting from zero knowledge to, well, some knowledge of the language. So if you, like me, are interested in trying out Julia language keep on reading. If you are already familiar with the subject you can safely skip this one.
+tags:
+ - Julia
+---
+
+## Let's try out Julia
```

Comment:
Taking a look at out blogs stats, Google visibility boosts posts immensely. Google puts heaviest emphasis on main topic of the page, as well as headings in general - in addition to links to it from external sites.

So thinking out loud, if you want a post that is easily found by organic google searches, topic is the most important thing. I don't have a great suggestion to what it could be here, but just a heads-up to think about it. Perhaps include work framework, or development? Title seems to include programming language, so that should help some already.